docs/snippets/alternative-work-generation-setup-preferred.md

```diff
@@ -0,0 +1,2 @@
+!!! warning "Alternative work generation setup preferred"
+	This option is not recommended for work generation for nodes due to the potential for performance impacts to the node participating on the network (voting, staying in sync, etc.). Please see the [Work Generation guide](/integration-guides/work-generation) for best practices.
```

Comment:
> This option is not recommended

I think we should make it more clear that the recommendation is to set to "0" in these cases, which is not the default.